crates/ruff/resources/test/fixtures/perflint/W8102.py

```diff
@@ -0,0 +1,32 @@
+some_dict = {
```

Comment:
Thanks for working on this. Can we add some more test cases?

```python
# Parenthesis
for (key, value) in some_dict.items():
	...

for (key, _) in some_dict.items():
	...

for (_, value) in some_dict.items():
	...

# Single variable with `items`
for pair in some_dict.items():
	...
```

There are bunch of real world cases which can be used for testing purposes: https://sourcegraph.com/search?q=context:global+lang:Python+for+%5C(_,+_%5C)+in+%5Cw+%5C.items&patternType=regexp&sm=1&groupBy=repo
* Key/value is a tuple where some elements are ignored
* Key/value is a tuple but using `(_, _)` to ignore instead of `_`

Reply:
Good catch thanks! I have added the extra test cases. Current implementation does not catch these yet so will find some time to come up with something clean to check for all three scenarios:
```
ExprName, ExprName
ExprName, ExprTuple
ExprTuple, ExprTuple
```

Reply:
New edits should catch these scenarios. I'll see if I can make it a bit less clunky and verbose.